mautic/templates/cache-clear-job.yaml

```diff
@@ -0,0 +1,28 @@
+{{- if .Values.cacheClear.enabled -}}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ include "mautic.fullname" $ }}-cache-clear
+  labels:
+    {{- include "mautic.labels" $ | nindent 4 }}
+  annotations:
+    "helm.sh/hook": post-install,post-upgrade
+    "helm.sh/hook-weight": "1"
+    "helm.sh/hook-delete-policy": hook-succeeded,hook-failed    
+spec:
+  template:
+    spec:
+      restartPolicy: Never
+      containers:
+        - name: {{ $.Chart.Name }}-cache-clear
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          command: ["/bin/sh", "-c", "php /var/www/html/bin/console cache:clear; php /var/www/html/bin/console cache:warmup; \
```

Comment:
Commands must be connected with `&&`. If one of the commands fails next shouln't run and container should fail to start.

As well setup permissions to all the `var/ cache` folder, not only for `prod` folder.
Environment can be different.